src/Caching/SynchedValue.purs

```diff
@@ -65,3 +69,9 @@ read (SynchedValue { avar }) = do
   Slot { value } <- AVar.read avar
   -- TODO check expiration time
   pure value
+
+-- | Check if a SynchedValue is initialized
+-- |
+-- | Try reads synchronously from a SychedValue. If it is empty returns False.
+isInitialized ∷ ∀  m a. MonadEffect m => SynchedValue a -> m Boolean
```

Comment:
can we rename this to `isEmpty` and flip the returned value?
Initialized suggests the initial state and the initial state of a `SynchedValue` is being empty.
I just think it's more self-documenting.